tyk-pro/scripts/bootstrap.sh

```diff
@@ -56,6 +81,15 @@ exit_reason(){
       ;;
     "6")
       echo "ERROR: Hostname should not contain port number or protocol"
+      exit $1
+      ;;
+    "7")
+      echo "An organisation with this name already exists, please delete it or disable bootstrapping"
+      exit $1
+      ;;
+    "8")
+      echo "An organisation with this cname already exists, please delete it or disable bootstrapping"
```

Comment:
One minor thing. I think, it might be worth mentioning `deleting from Mongo/Postgre` instead of `please delete it`. But if you think the error message is clear enough, just keep it as it is. 😃

Reply:
I agree your version sounds more explicit.

Reply:
Added in the next commit.

Reply:
Fixed.